src/components/Avatar/__test__/Avatar.test.tsx

```diff
@@ -1,9 +1,9 @@
+import "jest-dom/extend-expect"
```

Comment:
Can we create a setup file and move these common exports there too?

Reply:
That's a great suggestion, did see a ton of redundant imports, shifted them all under setup in jest config.